orchestrator-salt/src/main/resources/salt/salt/log4jpatch/init.sls

```diff
@@ -0,0 +1,10 @@
+{% if salt['pillar.get']('log4j_patch_enabled',False) == True %}
+run_log4j_patch:
+  cmd.script:
```

Comment:
I can't really remember why, but we don't use cmd.script. Instead we copy the file with `file.managed` and run it with `cmd.run`